### PR TITLE
Review base docs

### PR DESCRIPTION
**Pull request by SamMarkowitz**

### CODE REVIEW

File: content/io/cloudslang/base/json/get_keys.sl [hunk 1]
```diff
@@ -9,9 +9,9 @@
 # Retrieves keys from a JSON object at a given JSON path.
 #
 # Inputs:
-#   - json_input - JSON from which to retrieve keys
+#   - json_input - JSON from which to retrieve keys - Example: '{"k1": {"k2": {"k3":"v3"}}}'
 #   - json_path - path from which to retrieve key represented as a list of keys and/or indices.
```
Comment (gituser173) [inline content/io/cloudslang/base/json/get_keys.sl]: maybe also add an example for json_path? WDYT?

File: content/io/cloudslang/base/network/rest/http_client_delete.sl [hunk 2]
```diff
@@ -20,11 +20,11 @@
 #   - proxy_port - optional - proxy server port - Default: '8080'
 #   - proxy_username - optional - user name used when connecting to the proxy
 #   - proxy_password - optional - proxy server password associated with the <proxy_username> input value
-#   - connect_timeout - optional - time to wait for a connection to be established, in seconds - Default: '0' (infinite)
-#   - socket_timeout - optional - time to wait for data to be retrieved, in seconds - Default: '0' (infinite)
+#   - connect_timeout - optional - time in seconds to wait for a connection to be established - Default: '0' (infinite)
+#   - socket_timeout - optional - time in seconds to wait for data to be retrieved - Default: '0' (infinite)
```
Comment (gituser173) [inline content/io/cloudslang/base/network/rest/http_client_delete.sl]: time to wait(seconds)? WDYT?

File: content/io/cloudslang/base/os/linux/restart_service.sl [hunk 3]
```diff
@@ -6,21 +6,31 @@
 # http://www.apache.org/licenses/LICENSE-2.0
 #
 ####################################################
-# This flow restart remote Linux service using ssh
+# Restarts a remote Linux service using SSH.
```
Comment (gituser173) [inline content/io/cloudslang/base/os/linux/restart_service.sl]: "Restart a" maybe like in the description of content/io/cloudslang/base/os/linux/restart_server.sl
 flow? Or "Restarts a" but I think it should be consistent in all descriptions. WDYT?

File: test/io/cloudslang/base/files/read_from_file.inputs.yaml [hunk 4]
```diff
@@ -9,7 +9,7 @@
 testReadFromFileSuccess:
   inputs:
     - expected_read_text: hello
-    - expected_error_message: ""
+    - expected_message: reading done successfully
```
Comment (gituser173) [inline test/io/cloudslang/base/files/read_from_file.inputs.yaml]: "reading was successful"?

**Comment by gituser173** [comment]
made a few comments, otherwise it looks good :+1:

File: content/io/cloudslang/base/json/get_keys.sl (hunk 1 above)
Comment (SamMarkowitz) [inline content/io/cloudslang/base/json/get_keys.sl]: It's on the next line. I'll indent to make it clearer.

File: content/io/cloudslang/base/network/rest/http_client_delete.sl (hunk 2 above)
Comment (SamMarkowitz) [inline content/io/cloudslang/base/network/rest/http_client_delete.sl]: This one is all over the place and I've tried it out so many different ways that they all sound weird to me now. So I did some googling and it seems people do it a lot of different ways, but the way we have here seems to be widespread enough to leave it as is.

File: content/io/cloudslang/base/os/linux/restart_service.sl (hunk 3 above)
Comment (SamMarkowitz) [inline content/io/cloudslang/base/os/linux/restart_service.sl]: Yes. "Restarts" it is for both.

**Comment by SamMarkowitz** [comment]
Addressed the issues.

**Pull request merged**